### PR TITLE
Various RPC help outputs updated

### PR DESCRIPTION
**Pull request by Mirobit**
Typos, formatting and missing fields.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Concept ACK 34645c1

**Review by jonasschnelli — APPROVED** [review]
utACK

**Comment by laanwj** [comment]
Would prefer to postpone this to after named arguments (#8811). That should go in soon, and also updates many RPC help texts.

**Comment by Mirobit** [comment]
Rebased, #8811

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -1657,6 +1662,9 @@ UniValue listsinceblock(const JSONRPCRequest& request)
             "    \"txid\": \"transactionid\",  (string) The transaction id. Available for 'send' and 'receive' category of transactions.\n"
             "    \"time\": xxx,              (numeric) The transaction time in seconds since epoch (Jan 1 1970 GMT).\n"
             "    \"timereceived\": xxx,      (numeric) The time received in seconds since epoch (Jan 1 1970 GMT). Available for 'send' and 'receive' category of transactions.\n"
+            "    \"bip125-replaceable\": \"yes|no|unknown\",  (string) Whether this transaction could be replaced due to BIP125 (replace-by-fee);\n"
+            "                                                   may be unknown for unconfirmed transactions not in the mempool\n"
+            "    \"abandoned\": xxx          (bool) 'true' if the transaction has been abandoned (inputs are respendable). Only available for the 'send' category of transactions.\n"
```
Comment (jnewbery) [inline src/wallet/rpcwallet.cpp]: should have a comma since this isn't the final element in the returned object.

**Review by jnewbery — COMMENTED** [review]
A few minor nits.

File: src/wallet/rpcwallet.cpp (hunk 1 above)
Comment (Mirobit) [inline src/wallet/rpcwallet.cpp]: @jnewbery Thanks, fixed all nits.

**Comment by jnewbery** [comment]
utACK

File: src/rpc/blockchain.cpp [hunk 2]
```diff
@@ -336,8 +336,8 @@ std::string EntryDescriptionString()
            "    \"modifiedfee\" : n,      (numeric) transaction fee with fee deltas used for mining priority\n"
            "    \"time\" : n,             (numeric) local time transaction entered pool in seconds since 1 Jan 1970 GMT\n"
            "    \"height\" : n,           (numeric) block height when transaction entered pool\n"
-           "    \"startingpriority\" : n, (numeric) priority when transaction entered pool\n"
-           "    \"currentpriority\" : n,  (numeric) transaction priority now\n"
+           "    \"startingpriority\" : n, (numeric) DEPRECATED. priority when transaction entered pool\n"
```
Comment (paveljanik) [inline src/rpc/blockchain.cpp]: Small letters p and t (on the next line) look strange.

**Comment by paveljanik** [comment]
Concept ACK

File: src/rpc/blockchain.cpp (hunk 2 above)
Comment (Mirobit) [inline src/rpc/blockchain.cpp]: You are right, fixed it.

**Comment by MarcoFalke** [comment]
utACK 54ee3fc

**Pull request merged**